lockbox-ios/View/SettingListView.swift

```diff
@@ -157,6 +177,7 @@ extension SettingCellConfiguration: Equatable {
 class SwitchSettingCellConfiguration: SettingCellConfiguration {
     var isOn: Bool = false
 
+    var onChanged: AnyObserver<Bool>?
```

Comment:
🙌 I like this way better than the `switchChanged` method!!!

Is there a reason this is optional? Could you pass it into the initializer?